flink-connector-aws-kinesis-streams/src/main/java/org/apache/flink/connector/kinesis/sink/KinesisStreamsSinkWriter.java

```diff
@@ -144,6 +150,8 @@
                         .setMaxBufferedRequests(maxBufferedRequests)
                         .setMaxTimeInBufferMS(maxTimeInBufferMS)
                         .setMaxRecordSizeInBytes(maxRecordSizeInBytes)
+                        .setRateLimitingStrategy(
+                                buildRateLimitingStrategy(maxInFlightRequests, maxBatchSize))
```

Comment:
Since you did not change any unit tests this must be uncovered, or unasserted. Can you add tests to prevent regressions?

Reply:
yeah it was uncovered. 
thanks Danny will add Unit tests.